### ISSUE TITLE
coderopes-examples

### ISSUE THREAD

**Issue by ghost** [Dashboard-X/codropes-examples#1]
coderopes-examples

**Comment by dineshkummarc** [comment]
hi @denno-secqtinstien, what sort of enhancement is this ?

**Comment by denno-secqtinstien** [comment]
i was just updating some of the codes related but forgot that there is existing repo the same. i think you have updated the changes suggested by Neha-Choudhary

**Comment by denno-secqtinstien** [comment]
@dineshkummarc yes you have updated it i can see the issues with fixed label. thanks for update.
